eo-parser/src/test/resources/org/eolang/parser/packs/move-versions-down.yaml

```diff
@@ -0,0 +1,19 @@
+versioned: true
+tests:
```

Comment:
@maxonfjvipon In many cases here you are trying to find particular object, I guess. But it's hard to match `eo` code with `xpath`. You also use `name` attribute for some object. Maybe we can add `name` check for `xpath`'s? In this case it would help a reader to find an object.